iterations_runners/iterations_runner.som

```diff
@@ -0,0 +1,131 @@
+BenchmarkHarness = (
+
+    | benchmarkClass numIterations param debug instrument numCores |
+
+    krunInit = primitive
+    krunDone = primitive
+    krunGetNumCores = primitive
+    krunMeasure: idx = primitive
+    krunGetWallclock: idx = primitive
+    krunGetCoreCyclesDouble: index core: core = primitive
+
+    run: args = (
+        self processArguments: args.
+        self runBenchmark.
+    )
+
+    usage = (
+      'Iterations runner for SOM VMs.' println.
+      '' println.
+      'Executes a benchmark many times within a single process.' println.
+      '' println.
+      'usage: iterations_runner.som <benchmark> <# of iterations> <benchmark param>' println.
+      '<debug flag> [instrumentation dir] [key] [key pexec index]' println.
+      '' println.
+      'Arguments in [] are for instrumentation mode only.' println.
+      system exit.
+    )
+
+    processArguments: args = (
+        args length < 5 ifTrue: [ ^ self usage ].
+
+        self loadBenchmarkClass: (args at: 2). "First argument is the Benchmark"
+        numIterations := (args at: 3) asInteger.
+        param := (args at: 4) asInteger.
+        debug := (args at: 5) asInteger.
+
+        instrument := false.
+        args length >= 6 ifTrue: [
+            instrument := true.
+        ].
+
+        instrument && (args length ~= 8) ifTrue: [ ^ self usage ].
+    )
+
+    loadBenchmarkClass: className = (
+        | sym cls |
+        sym := className asSymbol.
+        cls := system load: sym.
+        cls println.
+        cls ifNil: [
+            self error: 'Failed loading benchmark: ', className ].
```

Comment:
I think `,` is better as `+` (see https://github.com/SOM-st/SOM/pull/56).

Reply:
Fixed: 5c0e33066356c2561d8281d72bb13f27bcbc829a